joomla/joomla-cms
# Link field for the full-article-image.

### Summary of Changes
This PR adds a modal link field that can be used to specify an url along with the title and rel attributes and uses it to provide an optional link for the full article image. This is very useful if a lightbox for images is used.
### Testing Instructions
1. Open the edit view of some article with a full article image in the backend.
2. Click on select next to the new field "Full image link"
3. Insert some values in the modal window.
4. Click save and also save the article.
5. Check if the article image in the frontend is now surrounded by the specified link with the according attributes.
### Documentation Changes Required
Add the new link field to the documentation.
